Doc/library/fractions.rst

```diff
@@ -84,6 +84,10 @@ another rational number, or from a string.
       The :class:`Fraction` constructor now accepts :class:`float` and
       :class:`decimal.Decimal` instances.
 
+   .. versionchanged:: 3.9
+      The :func:`math.gcd` function is now used to normalize the *numerator*
+      and *denominator*. :func:`math.gcd` always return a :class:`int` type.
+      Previously, the GCD type depended in *numerator* and *denominator*.
```

Comment:
Typo: "on" rather than "in"

Reply:
Oops, right, it's a typo. It's now fixed. Thanks for the review.